backend/benefit/companies/models.py

```diff
@@ -8,6 +9,7 @@ class Company(AbstractCompany):
     bank_account_number = IBANField(
         include_countries=("FI",), verbose_name=_("bank account number")
     )
+    department = CharField(verbose_name=_("department"), max_length=256, blank=True)
```

Comment:
Do we need a department field in Company model and where does the value for this field come from?
One company would have multiple departments, so is the department useful as a single charfield?
Maybe, as part of Yritysprofiili, we'd implement department info, but maybe we'd make a separate design for it later?

Reply:
Hmn, yeah I think you are right, I don't think we need the department field here. I'll fix it